nRF Toolbox/Profiles/HeartRateMonitor/Sections/Cells/LinearChartTableViewCell.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LinearChartTableViewCell.swift
+//  nRF Toolbox
+//
+//  Created by Nick Kibysh on 08/10/2019.
+//  Copyright © 2019 Nordic Semiconductor. All rights reserved.
+//
+
+import UIKit
+import Charts
+
+extension DateFormatter: IAxisValueFormatter {
+    public func stringForValue(_ value: Double,
+                        axis: AxisBase?) -> String {
+        let date = Date(timeIntervalSince1970: value)
+        return string(from: date)
+    }
+}
+
+class LinearChartTableViewCell: UITableViewCell {
+
+    let chartsView = LineChartView()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        contentView.addSubview(chartsView)
+        chartsView.legend.form = .empty
+        chartsView.noDataText = "Retrieving data..."
+        chartsView.noDataFont = .gtEestiDisplay(.thin, size: 32)
+        chartsView.noDataTextAlignment = .center
+        if #available(iOS 13, *) {
+            chartsView.noDataTextColor = .systemGray2
+        }
```

Comment:
TextColor? I noticed in the screenshots that in Dark Mode the text is black / difficult to read.